src/components/Grid/useGridSorting.js

```diff
@@ -22,27 +21,36 @@ export const useGridSorting = (rows, columns) => {
     return sortIcon
   }
 
+  const getSortDirection = (key) => {
+    if (currentSort[key] && currentSort[key].direction === 'up') {
+      return 'up'
```

Comment:
Maybe we should make `'up'`and `'down'` constants up at the top?

Reply:
commit 3dd316a39de7ad2d874c2e3ef47c46c396f95232 (HEAD -> useGridSorting-refactor)
Author: Rob Levin <rob@getethos.com>
Date:   Wed Jan 22 11:08:48 2020 -0800

    Use constant instead of magic string

Reply:
Pulled in 1.1.65 to Nora and verified:

![image](https://user-images.githubusercontent.com/142403/72925639-e2f51780-3d07-11ea-98ed-487088a1abb3.png)